strings/base_stringable_format.h

```diff
@@ -1,3 +1,8 @@
+inline std::wostream& operator<<(std::wostream& stream, winrt::Windows::Foundation::IStringable const& stringable)
```

Comment:
Use a different file for this. base_stringable_format is for std::format. base_stringable_streams?

Reply:
Done.